mlrun/common/schemas/workflow.py

```diff
@@ -32,6 +32,7 @@ class WorkflowSpec(pydantic.BaseModel):
     schedule: typing.Union[str, ScheduleCronTrigger] = None
     run_local: typing.Optional[bool] = None
     image: typing.Optional[str] = None
+    workflow_runner_node_selector: typing.Optional[dict] = None
```

Comment:
```suggestion
    workflow_runner_node_selector: typing.Optional[dict[str, str]] = None
```
Let's be consistent across the board.